test/over_react/component/_deprecated/abstract_transition_test.dart

```diff
@@ -753,6 +753,6 @@ class TransitionerComponent extends AbstractTransitionComponent<TransitionerProp
   @override
   void setState(dynamic newState, [Function() callback]) {
     super.setState(newState, callback);
-    transitionPhasesSet.add(newState.transitionPhase);
+    transitionPhasesSet.add(typedStateFactory(newState as Map).transitionPhase);
   }
```

Comment:
Good eye! It's because in `Component`, the arg is typed as `dynamic` (since it can take either a Map or an updater function), whereas in `Component2` we made it only accept `Map` and added `setStateWithUpdater`.

Reply:
ohh that makes sense!